pages/documentation/getting-started-developers/phase-two.md

```diff
@@ -69,62 +88,78 @@ In plain language, this code says:
 
 - **Create the foundation**: Build all USWDS styles from these settings.
 
-  The USWDS source code is the core of the design system. It contains all the styles for USWDS components as well as the design language of Sass tokens and functions used to build those components. USWDS source code has its own Sass entry point, which lives in the `node_modules` directory when you install USWDS with npm.
+  The USWDS source code is the core of the design system. It contains all the styles for USWDS components as well as the design language of Sass tokens and functions used to build those components. You will need to load USWDS source code after you define USWDS settings.
 
-  This is called `uswds` (or, more accurately, `uswds/_index.scss`), and it’s found in the `/packages` directory of the USWDS npm package. When you install with npm, the complete path is typically `./node_modules/@uswds/uswds/packages/uswds/_index.scss`.
+  USWDS source code has its own Sass entry point, which lives in the `node_modules` directory when you install USWDS with npm. This is called `uswds` (or, more accurately, `uswds/_index.scss`), and it’s found in the `/packages` directory of the USWDS npm package. When you install USWDS 3 with npm, the complete path is `./node_modules/@uswds/uswds/packages/uswds/_index.scss`.
 
 - **Build new work on top of that foundation**: Finally, add any custom project styles built from design system code.
 
-After you import the USWDS source code, you can build new styles with USWDS design tokens, functions, and mixins. For the purposes of this guide, we won’t get into custom code, but the important thing to understand is that any custom code should follow the settings and USWDS source code in your Sass entry point.
+  After you load the USWDS source code, you can build new styles with USWDS design tokens, functions, and mixins. For the purposes of this guide, we won’t get into custom code, but the important thing to understand is that any custom code should follow the settings and USWDS source code in your Sass entry point.
```

Comment:
Line 97: After you've loaded the USWDS source code (provide URL to documentation)

(the tense shift indicates that the step before this one should be completed, not that you need a whole new secret step)